### ISSUE TITLE
Unable to download dependencies on ./bin/gbuild

### ISSUE THREAD

**Issue by aigon89** [dashpay/dash#2143]
Hello,

I'm facing an error when I try to build the wallet using gitian, and is that when I use the ./bin/gbuild command, it looks like it's working fine, but it doesn't generate any executable.

Having a look at var/build.log, it shows that it is unable to fetch the dependencies needed,

(I add the .log file to the issue).

I've also tried to seed the gitan sources and offline git repositories with:

make -C ../dash/depends download SOURCES_PATH=`pwd`/cache/common

And after using the --url parameter, it didn't did the trick.

Has someone faced this issue?

Thank you all.

Best regards,

Aitor.

**Comment by PaulieD** [comment]
Is there any reason that you want to build using gitian instead of building it normally following the [instructions](https://github.com/dashpay/dash/blob/master/doc/build-unix.md)?

**Comment by schinzelh** [comment]
Looks like your build machine has issues resolving/downloading https://www.samba.org/ftp/ccache/ccache-3.3.3.tar.bz2 - maybe a connection/network issue on your side?

**Comment by aigon89** [comment]
no, not really, I was just playing. Even though, I believe it is the most convenient way if you wan to build multi pltform wallets.

@schinzelh I don't think so, I'm able to download it usin curl myself so I don't think it's a  network problem with the VM, but maybe a configuration problem regarding the lxc container it deplois for the build. ( I've had problems with it before like it not installing curl tools because those tools where not   a core package anymore and having to modify gitian-builder/bin/make-base-vm myself to include it myself, adding it to the addpkg variable, but I haven't seen any network parameters on the script so far.

Maybe it is useful to upload it.

Thank you so much!
[make-base-vm.zip](https://github.com/dashpay/dash/files/2147740/make-base-vm.zip)
